apps/acrs/translations/src/en/pages.json

```diff
@@ -142,7 +142,10 @@
     "header": "SKELETON: /partner-summary"
   },
   "children": {
-    "header": "SKELETON: /children"
+    "header": "Children aged under 18",
+    "paragraph-1": "You can refer your own children to come to the UK, but they must have been born on or after 28 August 2003.",
+    "paragraph-2": "The children you refer must not be living independently or have their own family.",
+    "question-1": "Are you referring your children to come to the UK?"
```

Comment:
I think in this case this text should be the 'legend' in the translations/fields file and should appear automatically without needing to render it in the view as a question.

See this: https://github.com/UKHomeOffice/acrs/blob/master/apps/acrs/fields/index.js#L228
and this: https://github.com/UKHomeOffice/acrs/blob/master/apps/acrs/translations/src/en/fields.json#L222

Reply:
Yes that's a better approach. I had tried a few combos but couldn't quite get it.